stdlib/log-api/src/main/ballerina/src/log/natives.bal

```diff
@@ -16,6 +16,30 @@
 
 import ballerina/java;
 
+# Represents the debug log level.
+public const DEBUG = "DEBUG";
+
+# Represents the error log level.
+public const ERROR = "ERROR";
+
+# Represents the info log level.
+public const INFO = "INFO";
+
+# Represents the trace log level.
+public const TRACE = "TRACE";
+
+# Represents the warn log level.
+public const WARN = "WARN";
+
+# Represents the all log level.
+public const ALL = "ALL";
+
+# Represents the off log level.
+public const OFF = "OFF";
+
+# Represents log level types.
+public type LogLevel DEBUG | INFO | ERROR | WARN | TRACE | OFF | ALL;
```

Comment:
Can use an enum for this. See https://github.com/ballerina-platform/ballerina-distribution/pull/611/files for examples on how to use enums.

Reply:
Updated in a15bfd2